frontend/src/metabase/collections/components/BaseItemsTable.jsx

```diff
@@ -139,6 +148,16 @@ function BaseItemsTable({
           }
         >
           <tr>
+            <ColumnHeader>
+              <EntityIconWrapper>
+                <CheckBox
+                  checked={selectedItems.length > 0}
+                  indeterminate={selectedItems.length > 0 && hasUnselected}
+                  onChange={hasUnselected ? onSelectAll : onSelectNone}
+                  data-testid="bulk-select"
+                />
```

Comment:
Should we consider using an `aria-label` here instead of a `data-testid` for a11y? That might also make our E2E tests a bit more readable:

```js
cy.findByTestId("bulk-select")
// vs
cy.findByLabelText("Select all items")
```

(FWIW, we shouldn't add a new unique user-facing string if it's targeting 46. So if it does, please feel free to ignore this comment or address it in another PR once the `release-x.x.x` branch is created)

Reply:
Yeah an `aria-label` definitely makes more sense, I can change this